test/testenv/deployment.go

```diff
@@ -252,6 +252,15 @@ func (d *Deployment) deployCR(name string, cr runtime.Object) (runtime.Object, e
 	return cr, nil
 }
 
+func (d *Deployment) updateCR(cr runtime.Object) error {
```

Comment:
Can we change the name to UpdateCR so that this method can be called outside of this file ?